### ISSUE TITLE
(Karen emailed Education Benefits and Applications Team PMs and PO) [L3: SCREENREADER]: VET TEC - NVDA does not read the error if I forget to agree to the privacy policy

### ISSUE THREAD

**Issue by taylorkaren** [department-of-veterans-affairs/va.gov-team#44852]

**Comment by joshkimux** [comment]
@artsymartha68 this issue has not had a response in a year. I've added education labels accordingly.
